docs/how-to/installation/docker.txt

```diff
@@ -56,29 +96,99 @@ master with the edited configuration:
 
 .. code::
 
-   docker run -v "$PWD"/master.yaml:/etc/determined/master.yaml determinedai/determined-master:VERSION
+   docker run \
+       -v "$PWD"/master.yaml:/etc/determined/master.yaml \
+       determinedai/determined-master:VERSION
+
+To start the master with environment variables instead of a configuration file:
+
+.. code::
+
+   docker run \
+       --name determined-master \
+       --network determined \
+       -p 127.0.0.1:8080:8080 \
+       -e DET_DB_HOST=determined-db \
+       -e DET_DB_NAME=determined \
+       -e DET_DB_PORT=5432 \
+       -e DET_DB_USER=postgres \
+       -e DET_DB_PASSWORD=<DB password> \
+       -e DET_HASURA_SECRET=<Hasura secret> \
+       -e DET_HASURA_ADDRESS=hasura:8080 \
+       determinedai/determined-master:VERSION
+
+Note that this references the PostgreSQL and Hasura containers running on
+``determined`` Docker network created above. If you are running PostgreSQL
+externally, specify the hostname for the PostgreSQL server in place of
+``determined-db``.
+
+Determined Agents
+~~~~~~~~~~~~~~~~~
+
+As is the case for the master, Determined agent :ref:`configuration values
+<cluster-configuration>` can come from a file, environment variables, or
+command-line arguments. To get a default configuration file for the agent,
+which contains a listing of the available options and descriptions for them,
+run:
+
+.. code::
+
+    id="$(docker create determinedai/determined-agent:VERSION)"
```

Comment:
blocking: The Docker image isn't actually built to contain a config file right now. That should probably be changed, though. Or just put in a link to the repo or something.

Reply:
I left this as is because it worked in my testing (for master and agent).  You don't think it should work?

Reply:
It's changed since the last release.

Reply:
Got it, changed to link to github rather than pull from our images